database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -46,8 +46,15 @@ trait MetadataEntryComponent {
     override def * = (workflowExecutionUuid, callFullyQualifiedName, jobIndex, jobAttempt, metadataKey, metadataValue,
       metadataValueType, metadataTimestamp, metadataEntryId.?) <> (MetadataEntry.tupled, MetadataEntry.unapply)
 
-    // TODO: rename index via liquibase
-    def ixMetadataEntryWeu = index("METADATA_WORKFLOW_IDX", workflowExecutionUuid, unique = false)
+    def pk = primaryKey("PK_METADATA_ENTRY", (metadataEntryId, workflowExecutionUuid))
+
+    def ixMetadataEntryUuidCallJobKey = index("METADATA_ENTRY_UUID_KEY_CALL_INDEX_ATTEMPT_INDEX",
+      (workflowExecutionUuid, callFullyQualifiedName, jobIndex, jobAttempt, metadataKey), unique = false
+    )
+
+    def ixMetadataEntryUuidKey = index("METADATA_ENTRY_UUID_KEY",
```

Comment:
Also my fault, but: based on other indexes elsewhere ([eg](https://github.com/broadinstitute/cromwell/blob/develop/database/migration/src/main/resources/changesets/mariadb_engine_schema.xml#L277)), I think this should be called `IX_METADATA_ENTRY_WEU_MK` (and in the changeset)